apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -19,142 +19,146 @@ import StandardsReport from '../sectionProgress/standards/StandardsReport';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
 import i18n from '@cdo/locale';
 
-class TeacherDashboard extends Component {
-  static propTypes = {
-    studioUrlPrefix: PropTypes.string.isRequired,
-    sectionId: PropTypes.number.isRequired,
-    sectionName: PropTypes.string.isRequired,
-    studentCount: PropTypes.number.isRequired,
-    coursesWithProgress: PropTypes.array.isRequired,
+function TeacherDashboard({
+  studioUrlPrefix,
+  sectionId,
+  sectionName,
+  studentCount,
+  coursesWithProgress,
+  location
+}) {
+  const usePrevious = value => {
+    const ref = React.useRef();
+    React.useEffect(() => {
```

Comment:
You should be able to just use `useEffect` here as you're importing it from `React` above

Reply:
Good catch- thanks!